config/manager/manager.yaml

```diff
@@ -69,7 +69,7 @@ spec:
         resources:
           limits:
             cpu: 500m
-            memory: 128Mi
+            memory: 512Mi
```

Comment:
Interesting! I guess running a `gRPC` is not free, huh?

Reply:
yea, I deployed this to the OpenShfit cluslter and hit the OOMKilled error. so I increased the memory. But I didn't really figure out how much memory the gRPC server increases.